index.html

```diff
@@ -0,0 +1,352 @@
+    <meta charset="UTF-8" />
+    <title>Enhance Your Skills GLOBAL Summit 2022</title>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+    />
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js"></script>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <nav
+      class="navbar nav1 navbar-expand-sm bg-dark navbar-dark justify-content-end py-0 p-3"
+    >
+      <ul class="navbar-nav">
+        <li class="nav-item">
+          <a class="nav-link" href="#">
+            <img src="./images/linke.png" alt="linkedin" />
+          </a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#">
+            <img src="./images/twitter.png" alt="twitter" />
+          </a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#">English</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#">My Page</a>
+        </li>
+        <li class="nav-item navLogout">
+          <a class="nav-link" href="#">Logout</a>
+        </li>
+      </ul>
+    </nav>
+
+    <!-- // navbar Section // -->
+
+    <nav class="navbar navbar-expand-sm bg-light navbar-light">
+      <div class="container">
+        <a class="navbar-brand" href="./index.html">
+          <img src="./images/Enha (1).png" alt="logo" id="logoMain" />
+        </a>
+        <button
+          class="navbar-toggler main_btn"
+          type="button"
+          data-bs-toggle="collapse"
+          data-bs-target="#collapsibleNavbar"
+        >
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div
+          class="collapse navbar-collapse justify-content-end"
+          id="collapsibleNavbar"
+        >
+          <ul id="menyOptions" class="navbar-nav nav2">
+            <li class="nav-item">
+              <a class="nav-link" href="./index.html">Home</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="./pages/about.html">About</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#spearkers">Speakers</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#partners">Partners</a>
+            </li>
+          </ul>
+          <ul id="popup" class="navbar-nav nav2">
+            <div id="cross-icon">
+              <!-- <img
+                onclick="off()"
+                class="style-li"
+                src="/images/close-Icon.png"
+                alt="close-Icon"
+              /> -->
```

Comment:
- [ ] Its good practice to remove commented code as this makes your codebase cleaner and easier to understand by other developers, kindly do so.